096-drop-travis-ci-and-testing-for-ppc-and-s390x.md

```diff
@@ -0,0 +1,63 @@
+# Drop Travis-CI and testing efforts for `ppc64le` and `s390x` architectures
+
+This proposal suggests to drop current usage of [Travis CI](https://www.travis-ci.com/) in [strimzi-kafka-bridge](https://github.com/strimzi/strimzi-kafka-bridge) and [strimzi-kafka-oauth](https://github.com/strimzi/strimzi-kafka-oauth) repositories.
+
+## Motivation
+
+In the past, we adopted Azure Pipelines as the primary CI system for the Operators repository, using it to run tests, build documentation, and create artifacts and images.
+The main reason for this transition was the improved resource quotas available for CNCF projects, which allowed us to run pipelines with far fewer restrictions than we had in Travis CI. 
+However, Azure Pipelines does not provide agents for the `ppc64le` and `s390x` architectures.
+This was not an issue for Operators, as we build multi-architecture images on `amd64` using _docker buildx_. 
+However, for Strimzi Kafka Bridge and Strimzi Kafka OAuth, we decided to retain the testing pipelines for `ppc64le` and `s390x`.
+
+In recent months, we have encountered several unexpected issues with Travis CI that required us to submit support tickets to restore our quota.
```

Comment:
does this mean the quotas ran out or were applied incorrectly?

Reply:
That's a good question. I think the quota on Travis CI has some kind of validity period in form of credits and it has to be manually renewed by Travis team from time to time, but I didn't find any order in that so I don't have proper answer.